portal-api/src/v1/facility-dates/cover-end-date.js

```diff
@@ -35,25 +22,40 @@ const hasAllCoverEndDateValues = (facility) => {
 
 exports.hasAllCoverEndDateValues = hasAllCoverEndDateValues;
 
+/**
+ * returns facility object with added coverEndDate property, if the day month,
+ * year are given. This is stored as a UTC timestamp
+ * @param {Object} facility 
+ * @returns {Object}
+ * 
+ * This function has odd behaviour inherited from moment js:
```

Comment:
The dependence on this behaviour wants investigating and removing (under a different ticket) if possible

Reply:
Yep -- Would raise ticket and add comment about this ticket (also feel this odd behaviour should be extracted into a date-helper so we don't have to document it everywhere)

Reply:
https://ukef-dtfs.atlassian.net/browse/DTFS2-7024